tests/transports-governance/test_governance_usage_tracking_hierarchical.py

```
@@ -0,0 +1,1001 @@
+        vk_id = vk_response.json()["id"]
+        track_created_entity("virtual_keys", vk_id)
+
+        # Simulate usage at VK level
+        usage_amount = 2000  # 20 dollars
+
+        # Update all budget levels to simulate hierarchical consumption
+        # In real implementation, this would be atomic
+        governance_client.update_budget(vk_budget_id, {"current_usage": usage_amount})
+        governance_client.update_budget(team_budget_id, {"current_usage": usage_amount})
+        governance_client.update_budget(
+            customer_budget_id, {"current_usage": usage_amount}
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)